### PR TITLE
Fixes #25541 - Trigger singleton actions in post init hook

### PR DESCRIPTION
**Pull request by adamruzicka**
requires:
- [x] https://github.com/Dynflow/dynflow/pull/307

### CODE REVIEW

**Comment by iNecas** [comment]
Tested and I can confirm this fixes the issue. One suggestion regrading removing the rake check + perhaps renaming the `post_init` (see the Dynflow PR)

File: lib/katello/engine.rb [hunk 1]
```diff
@@ -80,11 +80,9 @@ class Engine < ::Rails::Engine
     end
 
     initializer "katello.initialize_cp_listener", :before => :finisher_hook do
-      unless ForemanTasks.dynflow.config.remote? || File.basename($PROGRAM_NAME) == 'rake' || Rails.env.test?
```
Comment (iNecas) [inline lib/katello/engine.rb]: Shouldn't we keep the `Rails.env.test?` condition?

**Comment by jlsherrill** [comment]
looks good to me, i think we'll need a dynflow release for the tests to run

**Comment by adamruzicka** [comment]
[test katello]

**Pull request merged**